cmd/serverless-init/main.go

```diff
@@ -70,6 +70,7 @@ func setupTraceAgent(traceAgent *trace.ServerlessTraceAgent, tags map[string]str
 }
 
 func setupMetricAgent(tags map[string]string) *metrics.ServerlessMetricAgent {
+	config.Datadog.Set("use_v2_api.series", false)
```

Comment:
ah sorry I did not port this to `sls-init`

Reply:
Not your fault at all -- tbh maybe would be better if our shared code set these things, so we wouldn't miss it in the future. i.e move as many things outside of main.go in serverless and serverless-init and into pkg/serverless/metrics or something.